lab-maria/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "start": "node server.js",
+    "watch": "nodemon server.js",
+    "lint": ""
+  },
+  "author": "maria reed",
+  "license": "MIT",
+  "dependencies": {
+    "uuid": "^3.0.1"
+  }
```

Comment:
needs dev dependencies (chai) so when someone pulls your repo down from github, they can do an npm install and run your app without a hitch.